CONTRIBUTING.md

```diff
@@ -116,25 +117,25 @@ Finally, regenerate the documentation and generated code with `cargo dev generat
 
 Ruff's user-facing settings live in a few different places.
 
-First, the command-line options are defined via the `Cli` struct in `src/cli.rs`.
+First, the command-line options are defined via the `Cli` struct in `crates/ruff/src/cli.rs`.
 
-Second, the `pyproject.toml` options are defined in `src/settings/options.rs` (via the `Options`
-struct), `src/settings/configuration.rs` (via the `Configuration` struct), and `src/settings/mod.rs`
-(via the `Settings` struct). These represent, respectively: the schema used to parse the
-`pyproject.toml` file; an internal, intermediate representation; and the final, internal
-representation used to power Ruff.
+Second, the `pyproject.toml` options are defined in `crates/ruff/src/settings/options.rs` (via the
+`Options` struct), `crates/ruff/src/settings/configuration.rs` (via the `Configuration` struct), and
+`crates/ruff/src/settings/mod.rs` (via the `Settings` struct). These represent, respectively: the
+schema used to parse the `pyproject.toml` file; an internal, intermediate representation; and the
+final, internal representation used to power Ruff.
 
 To add a new configuration option, you'll likely want to modify these latter few files (along with
 `cli.rs`, if appropriate). If you want to pattern-match against an existing example, grep for
 `dummy_variable_rgx`, which defines a regular expression to match against acceptable unused
 variables (e.g., `_`).
 
 Note that plugin-specific configuration options are defined in their own modules (e.g.,
-`src/flake8_unused_arguments/settings.rs`).
+`crates/ruff/src/flake8_unused_arguments/settings.rs`).
 
 You may also want to add the new configuration option to the `flake8-to-ruff` tool, which is
 responsible for converting `flake8` configuration files to Ruff's TOML format. This logic
-lives in `flake8_to_ruff/src/converter.rs`.
+lives in `flake8_to_ruff/crates/ruff/src/converter.rs`.
```

Comment:
I think this is now be in `crates/ruff/src/flake8_to_ruff/converter.rs`. (The path listed here was incorrect, this got moved around.)